### DISCUSSION TITLE
Reworking the starter template

### DISCUSSION THREAD

**Discussion by DominusKelvin** [sailscastshq/boring-stack#12] [Ideas]
Currently, the starter template for The Boring JavaScript stack, is a throwaway counter app, which is not that useful. One of the philosophy of The Boring Stack is taking care of the boring stuff so you can be awesome.

That said, we need a new evolution of the starter template to have something useful and most of the features you will need for a full-stack application all setup for you, including testing of course and best practices.

**Comment by d-a-ve** [comment]
This is a great idea as having a simple counter app isn't enough starting ground for some people.

For the starter ideas, maybe a Todo app if you are going for something beginner friendly. 

Or an e-commerce or blog web app for intermediate to advanced level, depending on how large the app is and what functionalities will be added.

By doing this, I believe it becomes easy to incorporate the millow-starters that is currently being developed.
